daemon/mgr/container.go

```diff
@@ -360,6 +360,12 @@ func (mgr *ContainerManager) Create(ctx context.Context, name string, config *ty
 		container.Snapshotter.Data["UpperDir"] = upperDir
 	}
 
+	// validate container Config
+	warnings, err := validateConfig(config)
```

Comment:
I think this part validation could be located in the API bridge layer. WDYT? @Ace-Tang @fuweid

Reply:
@allencloud the config has changed by `ContainerManager).Create`, like 

```
if config.HostConfig.EnableLxcfs && lxcfs.IsLxcfsEnabled {
 		config.HostConfig.Binds = append(config.HostConfig.Binds, lxcfs.LxcfsParentDir+":/var/lib/lxc:shared")
 		sourceDir := lxcfs.LxcfsHomeDir + "/proc/"
 		destDir := "/proc/"
 		for _, procFile := range lxcfs.LxcfsProcFiles {
 			bind := fmt.Sprintf("%s%s:%s%s", sourceDir, procFile, destDir, procFile)
 			config.HostConfig.Binds = append(config.HostConfig.Binds, bind)
 		}
 	}
```

I don't think we can do validation in the API bridge level.

Reply:
@fuweid ,do you mean config is merged with something after `ContainerManager).Create`?

Reply:
Yes @Ace-Tang . If we can move the merge function into bridge, the validation can be done in bridge, too.